GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -1063,7 +1070,8 @@ private void DiffText_TextLoaded(object sender, EventArgs e)
         /// <returns>If 'artificial' handling</returns>
         private bool IsWorkTreeOrIndexWithParent()
         {
-            if (DiffFiles.SelectedItem == null ||
+            var item = DiffFiles.SelectedItem;
+            if (item == null ||
```

Comment:
Nit: This modification is unnecessary.

Reply:
It is an effect of my backported changes. I can revert it...

Reply:
Keep `item` if you are going to use it in a follow up PR.

Reply:
I reverted that, there will be other changes to the assignment